Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -103,189 +93,12 @@ void array_set_zero(float *data, size_t num_elem) {
-    }
-  }
-
-fail_exit:
-  status = ml_single_close(single);
-
-  return status;
-}
-#else
-/**
- * @brief Private data for Tensorflow lite object
- */
-struct TFLiteData {
-  tflite::ops::builtin::BuiltinOpResolver resolver;
-  std::unique_ptr<tflite::Interpreter> interpreter;
-  std::unique_ptr<tflite::FlatBufferModel> model;
-  std::string data_path;
-
-  int output_number_of_pixels;
-  int inputDimReq[MAX_DIM];
-};
-
-/**
- * @brief Load the tensorflow lite model and its metadata
- */
-void setupTensorflowLiteModel(const std::string &data_path,
-                              TFLiteData &tflite_data) {
-  int input_size;
-  int output_size;
-  int len;
-  int outputDim[MAX_DIM];
-
-  tflite_data.data_path = data_path;
-  std::string model_path =
-    data_path + "/" + "ssd_mobilenet_v2_coco_feature.tflite";
-  tflite_data.model =
-    tflite::FlatBufferModel::BuildFromFile(model_path.c_str());
-  if (tflite_data.model == NULL)
-    throw std::runtime_error("Unable to build model from file");
-
-  tflite::InterpreterBuilder(*tflite_data.model.get(),
-                             tflite_data.resolver)(&tflite_data.interpreter);
-
-  if (tflite_data.interpreter->AllocateTensors() != kTfLiteOk)
-    throw std::runtime_error("Failed to allocate tensors!");
-
-  input_size = tflite_data.interpreter->inputs().size();
-  output_size = tflite_data.interpreter->outputs().size();
-
-  if (input_size > 1 || output_size > 1)
-    throw std::runtime_error("Model is expected with single input and output");
-
-  for (int i = 0; i < MAX_DIM; i++) {
-    tflite_data.inputDimReq[i] = 1;
-    outputDim[i] = 1;
-  }
-
-  int input_idx = tflite_data.interpreter->inputs()[0];
-  len = tflite_data.interpreter->tensor(input_idx)->dims->size;
-  std::reverse_copy(tflite_data.interpreter->tensor(input_idx)->dims->data,
-                    tflite_data.interpreter->tensor(input_idx)->dims->data +
-                      len,
-                    tflite_data.inputDimReq);
-
-  int output_idx = tflite_data.interpreter->outputs()[0];
-  len = tflite_data.interpreter->tensor(output_idx)->dims->size;
-  std::reverse_copy(
-    tflite_data.interpreter->tensor(output_idx)->dims->data,
-    tflite_data.interpreter->tensor(output_idx)->dims->data + len, outputDim);
-
-  tflite_data.output_number_of_pixels = 1;
-  for (int k = 0; k < MAX_DIM; k++)
-    tflite_data.output_number_of_pixels *= tflite_data.inputDimReq[k];
-}
-
 /**
- * @brief     Get Feature vector from tensorflow lite
- *            This creates interpreter & inference with ssd tflite
+ * @brief     Load input image data from bmp files and normalize to float
  * @param[in] filename input file path
- * @param[out] feature_input save output of tflite
+ * @param[out] feature_input normalized images
  */
-void getInputFeature(const TFLiteData &tflite_data, const std::string filename,
-                     float *feature_input) {
+void getInputFeature(const std::string filename, float *feature_input) {
```

Comment:
Since some people are referring to the code...
```
void getInputFeature(const std::string &filename, float *feature_input) {
```

Reply:
Will be done.